lib/galaxy/jobs/__init__.py

```diff
@@ -1402,7 +1402,7 @@ def _collect_metrics( self, has_metrics ):
         job = has_metrics.get_job()
         per_plugin_properties = self.app.job_metrics.collect_properties( job.destination_id, self.job_id, self.working_directory )
         if per_plugin_properties:
-            log.info( "Collecting job metrics for %s" % has_metrics )
+            log.info( "Collecting job metrics for %s: %s" % ( has_metrics, getattr( has_metrics, 'id', None ) ) )
```

Comment:
@blankenberg I understand, in fact in the example above I proposed to remove "job" from the string and add `type(has_metrics)`.